### PR TITLE
rename 'clusters' in Cloud Environments page

### PR DESCRIPTION
**Pull request by panentheos**
This is a subset of the overall rename effort.

### CODE REVIEW

File: src/pages/Clusters.js [hunk 1]
```diff
@@ -132,27 +132,27 @@ const Clusters = () => {
     size: 'size'
   }[diskSort.field]], [diskSort.direction], disks)
 
-  const totalCost = _.sum(_.map(clusterCost, clusters))
+  const totalCost = _.sum(_.map(clusterCost, runtimes))
```
Comment (s-rubenstein) [inline src/pages/Clusters.js:135]: Presumably clusterCost is going to be changed with `cluster-utils`?

File: src/pages/Clusters.js [hunk 2]
```diff
@@ -132,27 +132,27 @@ const Clusters = () => {
...
         columns: [
           {
             headerRenderer: () => h(Sortable, { sort, field: 'project', onSort: setSort }, ['Billing project']),
             cellRenderer: ({ rowIndex }) => {
-              const cluster = filteredClusters[rowIndex]
-              const inactive = !_.includes(cluster.status, ['Deleting', 'Error']) &&
-                currentCluster(clustersByProject[cluster.googleProject]) !== cluster
+              const runtime = filteredRuntimes[rowIndex]
+              const inactive = !_.includes(runtime.status, ['Deleting', 'Error']) &&
+                currentCluster(runtimesByProject[runtime.googleProject]) !== runtime
```
Comment (s-rubenstein) [inline src/pages/Clusters.js:153]: Presumably currentCluster is going to change with `cluster-utils`?

**Review by s-rubenstein — COMMENTED** [review]
Nit: We should probably also rename the file to Environments.js? And then could we put a note above line 345 saying that the path is to maintain backwards compatibility with old urls?

**Comment by panentheos** [comment]
File renames will be done in a separate PR. If you make substantial changes to the contents of a file at the same time as renaming it, git sometimes fails to classify it as a rename, which can make archaeology harder.

File: src/pages/Clusters.js (hunk 1 above)
Comment (panentheos) [reply src/pages/Clusters.js:135]: Yes, this PR is scoped to just local names within one file for easier reviewing.

**Review by s-rubenstein — APPROVED** [review]
LGTM

**Pull request merged**